comm/src/main/scala/coop/rchain/comm/comm.scala

```diff
@@ -16,8 +16,8 @@ case class NodeIdentifier(pKey: Seq[Byte]) {
 }
 
 case class Endpoint(host: String, tcpPort: Int, udpPort: Int) {
-  lazy val tcpSocketAddress = new java.net.InetSocketAddress(host, tcpPort)
```

Comment:
Why did you get rid of the lazys?

Reply:
Deferring to superior Scala wisdom: https://github.com/rchain/rchain/pull/22#discussion_r145754028